### PR TITLE
[Proc] Add two specs to check SyntaxError for block arguments

### PR DESCRIPTION
**Pull request by JuanitoFatas**
This pull request adds two specs for block arguments syntax check.

--

Learned from here: https://github.com/thoughtbot/guides/pull/335#issuecomment-112882320

### CODE REVIEW

**Comment by eregon** [comment]
Looks good, but this should be in `language/` since it raises a SyntaxError.
It might be worth having it for method parameters as well.

---
There is also some interesting behavior with this doubled `_`:
```ruby
p lambda {|_,_|_}.(1,2) # => 1
```
But I would not consider that spec.

**Comment by eregon** [comment]
(feel free to force push here or make another PR to move the spec to `language`)

**Comment by JuanitoFatas** [comment]
> (feel free to force push here or make another PR to move the spec to language)

:ok_hand: Will do it later today, thank you Benoit :+1:

**Comment by JuanitoFatas** [comment]
@eregon Please take a look, thanks!

File: language/block_spec.rb [hunk 1]
```diff
@@ -634,6 +634,36 @@ def m(a) yield a end
   end
 end
 
+describe "block arguments" do
```
Comment (eregon) [inline language/block_spec.rb]: Block

**Comment by eregon** [comment]
@JuanitoFatas Thanks for you contribution!
I have a couple comments but I think they will expressed more clearly in an additional commit.

**Comment by eregon** [comment]
Have a look at ea9f106c4bc37abed03d66c70511c495d17274dc.

**Comment by JuanitoFatas** [comment]
Much better! Thanks @eregon :clap:

**Pull request merged**